File: PartyMemberManagementSystem/GUI/Security/FormUserSetting.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using PartyMemberManagementSystem.Properties;

namespace PartyMemberManagementSystem.GUI.Security
{
    public partial class FormUserSetting : Form
    {
        public FormUserSetting()
        {
            InitializeComponent();
        }

        private void txtDataGridViewAlternativeColor_MouseClick(object sender, MouseEventArgs e)
        {
            ColorDialog cd = new ColorDialog();
            cd.Color = this.txtDataGridViewAlternativeColor.BackColor;
            if ((cd.ShowDialog() == DialogResult.OK))
            {
                this.txtDataGridViewAlternativeColor.BackColor = cd.Color;
            }
        }

        private void btnSave_Click(object sender, EventArgs e)
        {
            Settings.Default.DataGridViewAlternativeColor = this.txtDataGridViewAlternativeColor.BackColor;
            Settings.Default.DataGridViweRowHeight = Int32.Parse(txtDataGridViewRowHeigt.Text);
            Settings.Default.DataGridViweFont = this.txtDataGridViwFontName.Font;
            Settings.Default.Languguage = this.cboLanguage.Text;
            Settings.Default.Save();
            this.Close();
        }

        private void FormUserSetting_Load(object sender, EventArgs e)
        {
            txtDataGridViewAlternativeColor.BackColor = Settings.Default.DataGridViewAlternativeColor;
            txtDataGridViewRowHeigt.Text = Settings.Default.DataGridViweRowHeight.ToString();
            txtDataGridViwFontName.Font = Settings.Default.DataGridViweFont;
            cboLanguage.Text = Settings.Default.Languguage;
        }

        private void txtDataGridViwFontName_MouseClick(object sender, MouseEventArgs e)
        {
            FontDialog fd = new FontDialog();
            fd.Font = this.txtDataGridViwFontName.Font;
            if (fd.ShowDialog() == DialogResult.OK)
            {
                this.txtDataGridViwFontName.Font = fd.Font;
            }
        }
    }
}

File: SaleApplication/Forms/System/FormUserSetting.vb
Public Class FormUserSetting

    Private Sub Button2_Click(ByVal sender As System.Object, ByVal e As System.EventArgs)

    End Sub

    Private Sub txtDataGridViewAlternativeColor_MouseClick(ByVal sender As System.Object, ByVal e As System.Windows.Forms.MouseEventArgs) Handles txtDataGridViewAlternativeColor.MouseClick
        Dim cd As New ColorDialog()
        cd.Color = Me.txtDataGridViewAlternativeColor.BackColor
        If (cd.ShowDialog() = Windows.Forms.DialogResult.OK) Then
            Me.txtDataGridViewAlternativeColor.BackColor = cd.Color
        End If
    End Sub

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click

        My.Settings.DataGridViewAlternativeColor = Me.txtDataGridViewAlternativeColor.BackColor
        My.Settings.DataGridViweRowHeight = Me.txtDataGridViewRowHeigt.Text
        My.Settings.DataGridViweFont = Me.txtDataGridViwFontName.Font
        My.Settings.Languguage = Me.cboLanguage.Text
        My.Settings.Save()
        Me.Close()
    End Sub

    Private Sub FormUserSetting_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        Me.txtDataGridViewAlternativeColor.BackColor = My.Settings.DataGridViewAlternativeColor
        Me.txtDataGridViewRowHeigt.Text = My.Settings.DataGridViweRowHeight
        Me.txtDataGridViwFontName.Font = My.Settings.DataGridViweFont
        Me.cboLanguage.Text = My.Settings.Languguage 
    End Sub

    Private Sub txtDataGridViwFontName_MouseClick(ByVal sender As System.Object, ByVal e As System.Windows.Forms.MouseEventArgs) Handles txtDataGridViwFontName.MouseClick
        Dim fd As New FontDialog()
        fd.Font = Me.txtDataGridViwFontName.Font
        If fd.ShowDialog() = Windows.Forms.DialogResult.OK Then
            Me.txtDataGridViwFontName.Font = fd.Font
        End If
    End Sub
 
End Class